airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -94,6 +98,45 @@ definitions:
       message:
         description: "the log message"
         type: string
+  AirbyteTraceMessage:
+    type: object
+    additionalProperties: true
+    required:
+      - type
+      - emitted_at
+    properties:
+      type:
+        description: "the type of trace message"
+        type: string
+        enum:
+          - ERROR
+      emitted_at:
+        description: "the time in ms that the message was emitted"
+        type: number
+      error:
+        description: "error data from the sync"
+        "$ref": "#/definitions/AirbyteErrorTraceMessage"
+  AirbyteErrorTraceMessage:
+    type: object
+    additionalProperties: true
```

Comment:
👍 on `additionalProperties` - The thought was that we might take any additional properties and append them to the FailureReason's "metadata" property. 

cc @pedroslopez

Reply:
recap from discussion on the doc: this will not be the case - `additionalProperties` is `true` so that it will be ok if a connector does erroneously set an additional property, and it's easier to make backwards-compatible changes to the protocol later. 

No further properties are expected to be included in `AirbyteErrorTraceMessage` and the connector logs should give additional context.